CONTRIBUTING.md

```diff
@@ -70,21 +70,21 @@ move forward. See editor Lee Byron talk about
 * **Enable new capabilities motivated by real use cases**
 
   Every change should intend on unlocking a real and reasonable use case. Real
-  examples are always more interesting than theoretical ones, and common
-  scenarios are more interesting than rare ones. RFCs should do more than offer
+  examples are always more compelling than theoretical ones, and common
+  scenarios are more compelling than rare ones. RFCs should do more than offer
   a different way to reach an already achievable outcome.
 
 * **Simplicity and consistency over expressiveness and terseness**
 
-  There are plenty of behaviors and patterns found in other languages
+  There are plenty of behaviors and patterns found in other languages that are
```

Comment:
I think this is fine with or without `that are`; without it is the more condensed form, in general I try to avoid putting the word `that` into my prose as it's rarely necessary.

https://english.stackexchange.com/questions/50962/what-is-a-noun-modifying-clause/50971#50971
```suggestion
  There are plenty of behaviors and patterns found in other languages
```

Reply:
I switched around the sentence to avoid awkwardness without "that".